src/server/auth/server/testing/config_test.go

```diff
@@ -1,813 +0,0 @@
-package server
-
-import (
-	"encoding/xml"
-	"fmt"
-	"testing"
-	"time"
-
-	"github.com/gogo/protobuf/proto"
-
-	"github.com/crewjam/saml" // used to format saml IdP in config
-	"github.com/pachyderm/pachyderm/src/client/auth"
-	"github.com/pachyderm/pachyderm/src/client/pkg/errors"
-	"github.com/pachyderm/pachyderm/src/client/pkg/require"
-	authserver "github.com/pachyderm/pachyderm/src/server/auth/server"
-	"github.com/pachyderm/pachyderm/src/server/pkg/backoff"
-	tu "github.com/pachyderm/pachyderm/src/server/pkg/testutil"
-)
-
-// requireConfigsEqual compares 'expected' and 'actual' using 'proto.Equal', but
-// also formats any XML they contain to remove e.g. whitespace discrepancies
-// that we want to ignore
-func requireConfigsEqual(t testing.TB, expected, actual *auth.AuthConfig) {
-	e, a := proto.Clone(expected).(*auth.AuthConfig), proto.Clone(actual).(*auth.AuthConfig)
-	// Format IdP metadata (which is serialized XML) to fix e.g. whitespace
-	formatIDP := func(c *auth.IDProvider) {
-		if c.SAML == nil || c.SAML.MetadataXML == nil {
-			return
-		}
-		var d saml.EntityDescriptor
-		require.NoError(t, xml.Unmarshal(c.SAML.MetadataXML, &d))
-		c.SAML.MetadataXML = tu.MustMarshalXML(t, &d)
-	}
-	for _, idp := range e.IDProviders {
-		formatIDP(idp)
-	}
-	for _, idp := range a.IDProviders {
-		formatIDP(idp)
-	}
-	require.True(t, proto.Equal(e, a),
-		"expected: %v\n           actual: %v", e, a)
-}
-
-// TestSetGetConfigBasic sets an auth config and then retrieves it, to make
-// sure it's stored propertly
-func TestSetGetConfigBasic(t *testing.T) {
```

Comment:
IIUC, we'll still have a `SetConfiguration` endpoint—would it make sense to at least keep around this test for it? (or is there like another test elsewhere?)

Reply:
Reinstated `config_test.go` and removed all but the few tests that were still relevant.